### PR TITLE
10 documentation and specs for select

### PR DESCRIPTION
**Pull request by zth96**
I added select documentation. Please double check. Thanks.

### CODE REVIEW

File: src/tidyversetopandas/tidyversetopandas.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    columns (list of str): Column names to select.
+
+    Returns:
+    pdf.DataFrame: A DataFrame with only the selected columns.
+
+    Example DataFrame:
+    >>> data = {"column1': [1, 2, 3], 'column2': [4, 5, 6], 'column3': [7, 8, 9]}
+    >>> df = pd.DataFrame(data)
+    """
+    return dataframe[list(columns)]
```
Comment (farrandi) [inline src/tidyversetopandas/tidyversetopandas.py:19]: It is typical to add an extra line at the end of document
```suggestion
    return dataframe[list(columns)]
    
```

**Review by farrandi — COMMENTED** [review]
Looks good!

Just gave a minor nit.

File: src/tidyversetopandas/tidyversetopandas.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from tidyversetopandas import tidyversetopandas
```
Comment (farrandi) [inline src/tidyversetopandas/tidyversetopandas.py:1]: Also you do not need this line.

```suggestion
```

File: src/tidyversetopandas/tidyversetopandas.py (hunk 1 above)
Comment (zth96) [reply src/tidyversetopandas/tidyversetopandas.py:19]: Will do thanks!

File: src/tidyversetopandas/tidyversetopandas.py (hunk 2 above)
Comment (zth96) [reply src/tidyversetopandas/tidyversetopandas.py:1]: Sure thanks

**Review by farrandi — APPROVED** [review]
LGTM

**Review by LilyTao0531 — APPROVED** [review]

File: src/tidyversetopandas/tidyversetopandas.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    dataframe (pd.DataFrame): The DataFrame to select columns from.
+    columns (list of str): Column names to select.
+
+    Returns:
+    pdf.DataFrame: A DataFrame with only the selected columns.
+
+    Example DataFrame:
+    >>> data = {"column1': [1, 2, 3], 'column2': [4, 5, 6], 'column3': [7, 8, 9]}
+    >>> df = pd.DataFrame(data)
+    """
```
Comment (786213750) [inline src/tidyversetopandas/tidyversetopandas.py:17]: you can add a example for the function
```suggestion
    select(df, "column1", "column2")
    """
```

**Review by 786213750 — APPROVED** [review]
everything looks good

**Pull request merged**